TestSuite/pack.py

```diff
@@ -186,9 +203,14 @@ def create_incident_field(
             self,
             name,
             content: dict = None,
+            release_notes: str = False
```

Comment:
str = false?

Reply:
Sorry forgot to change that. doing it now